content/en/blog/_posts/grokkin-the-docs.md

```diff
@@ -0,0 +1,327 @@
+- Does the PR follow the guidelines in the style and content guides?
+
+  - Overall, is the grammar and punctuation correct?
+  - Is the content clear, concise, and appropriate for non-native speakers?
+  - Does the content stylistically fit in with the rest of the documentation?
+  - Does the flow of the content make sense?  
+  - Can anything be changed to make the content better, such as using a Hugo shortcode?
+  - Does the content render correctly?
+
+- Is the content technically correct?
+
+Sometimes the review process made me feel defensive, annoyed, and frustrated. I'm
+sure other contributors have felt the same way. Contributors need to be patient!
+Writing excellent documentation is an iterative process. Reviewers scrutinize
+PRs because they want to maintain a high level of quality in the documentation,
+not because they want to annoy contributors!
+
+## Observation 06: Make every word count {#observation-6}
+
+Non-native English speakers read and contribute to the Kubernetes documentation.
+When you are writing content, use simple, direct language in clear, concise
+sentences.  Every sentence you write may be translated into another language, so
+remove words that don't add substance. I admit that implementing these
+guidelines is challenging at times.
+
+Issues and pull requests aren't translated into other languages. However, you
+should still follow the aforementioned guidelines when you write the description
+for an issue or pull request. You should add details and background
+information to an issue so the person doing triage doesn't have to apply the
+`triage/needs-information` label. Likewise, when you create a pull request, you
+should add enough information about the content change that reviewers don't have
+to figure out the reason for the pull request. Providing details in clear,
+concise language speeds up the process.
+
+## Observation 07: Triaging issues is more difficult than it should be {#observation-7}
+
+In SIG Docs, triaging issues requires the ability to distinguish between
+support, bug, and feature requests not only for the documentation but also for
+Kubernetes code projects. How to route, label, and prioritize issues has become
+easier week by week. I'm still not 100% clear on which SIG and/or project is
+responsible for which parts of the documentation. The SIGs and Working Groups
+[page](https://github.com/kubernetes/community/blob/master/sig-list.md) helps,
+but it is not enough. At a page level in the documentation, it's not
+always obvious which SIG or project has domain expertise. The page's front
+matter sometimes list reviewers but never lists a SIG or project. Each page should
+indicate who is responsible for content, so that SIG Docs triagers know where to
+route issues.
+
+## Observation 08: SIG Docs is understaffed {#observation-8}
+
+Documentation is the #1 driver of software adoption<sup>1</sup>.
+
+Yet few companies are willing to allocate tech writers at least
```

Comment:
Hopefully they _are_ willing!

```suggestion
Yet few companies have hired tech writers at least
```

Reply:
For the life of me I could not remember what you replaced this paragraph with during our meeting.

Reply:
I can't remember, either. 😞

Reply:
I did remove the "Of people..." paragraphs. Let me know what you think.